build.conf

```diff
@@ -36,19 +38,16 @@ SQLITE_VERSION="3460000"
 BOOTSTRAP_GET_PIP="https://bootstrap.pypa.io/get-pip.py"
 
 # Python modules installed after bootstraping pip.
-PIP_VERSION="24.0"
-SETUPTOOLS_VERSION="70.0.0"
+PIP_VERSION="24.2"
+SETUPTOOLS_VERSION="75.1.0"
```

Comment:
Do we need the newer setup tools version ?

If possible, it's best to keep the older version.

The problem is that setuptools comes with it's subset of vendored packages... and these vendored packages are advertised to pip ... and pip is confused

------

Do we need setuptools at all in pythia ?

Reply:
What would be the final resolution for this? Keep `setuptools` when packing up for now? Or uninstall it here, and then make sure it's installed in server and other repos as needed?

Reply:
This was discussed further down, then added as #66.